tests/test_lookml.py

```diff
@@ -0,0 +1,303 @@
+                                fields=[
+                                    bigquery.schema.SchemaField("country", "STRING"),
+                                ],
+                            ),
+                            bigquery.schema.SchemaField(
+                                "header",
+                                "RECORD",
+                                fields=[
+                                    bigquery.schema.SchemaField("date", "STRING"),
+                                    bigquery.schema.SchemaField(
+                                        "parsed_date", "TIMESTAMP"
+                                    ),
+                                ],
+                            ),
+                        ],
+                    ),
+                    bigquery.schema.SchemaField("parsed_timestamp", "TIMESTAMP"),
+                    bigquery.schema.SchemaField("submission_timestamp", "TIMESTAMP"),
+                    bigquery.schema.SchemaField("test_bignumeric", "BIGNUMERIC"),
+                    bigquery.schema.SchemaField("test_bool", "BOOLEAN"),
+                    bigquery.schema.SchemaField("test_bytes", "BYTES"),
+                    bigquery.schema.SchemaField("test_float64", "FLOAT"),
+                    bigquery.schema.SchemaField("test_int64", "INTEGER"),
+                    bigquery.schema.SchemaField("test_numeric", "NUMERIC"),
+                    bigquery.schema.SchemaField("test_string", "STRING"),
+                ],
+            )
+        raise ValueError(f"Table not found: {table_ref}")
+
+
+def test_lookml(runner, namespaces):
+    with runner.isolated_filesystem():
+        with patch("google.cloud.bigquery.Client", MockClient):
+            result = runner.invoke(
+                lookml,
+                [
+                    "--namespaces",
+                    namespaces,
+                ],
+            )
+        sys.stdout.write(result.stdout)
+        if result.stderr_bytes is not None:
+            sys.stderr.write(result.stderr)
+        try:
+            assert not result.exception
+        except Exception:
+            traceback.print_tb(result.exc_info[2])
+            raise
+        assert (
+            dedent(
+                """
+            view: baseline {
+              sql_table_name: `mozdata.custom.baseline` ;;
+
+              dimension: client_id {
+                hidden: yes
+                sql: ${TABLE}.client_id ;;
+              }
+
+              dimension: country {
+                map_layer_name: countries
+                sql: ${TABLE}.country ;;
+                type: string
+              }
+
+              dimension: document_id {
+                hidden: yes
+                sql: ${TABLE}.document_id ;;
+              }
+
+              measure: clients {
+                sql: COUNT(DISTINCT client_id) ;;
+                type: number
+              }
+
+              measure: count {
+                type: count
+              }
```

Comment:
We should drop these in favor of `ping_count` as `type: count`

Reply:
looker auto-generates:
```lookml
  measure: count {
    type: count
    drill_fields: []
  }
```
And I think it's worth sticking to that until we differentiate between stable and derived tables, because this generates views for `baseline_clients_daily` and `baseline_clients_last_seen` tables right now, and `count` is definitely not `ping_count` there.

Reply:
I agree, we should only generate this field for stable table views. I'm happy with updating this as soon as we can differentiate the two. However I don't necessarily think we should generate a count field for _daily or _last_seen tables either, until we agree on a general structure for representing those.

Reply:
I modified this to do `ping_count` when `document_id` is present, and `clients` when `client_id` is present, and only add `count` if neither of those were found.